/**:
    ros__parameters:
      app_name: "YourPSDK"
      app_id: "YourID"
      app_key: "YourAppKey"
      app_license: "YourAppLicense"
      developer_account: "YourDeveloperAccount"
      baudrate: "921600"

      num_of_initialization_retries: 3

      imu_frame: "psdk_imu_link"
      body_frame: "psdk_base_link"
      map_frame: "psdk_map_enu"
      gimbal_frame: "psdk_gimbal_link"
      camera_frame: "psdk_camera_link"
      publish_transforms: true

      # Mandatory modules to be initialized. Mark with a true those which you
      # consider mandatory for your application, false otherwise. Be aware that
      # some modules might have inter-dependencies. Non mandatory modules will
      # still be initialized but if case of failure, the node will continue to run.
      mandatory_modules:
        telemetry: true
        flight_control: true
        camera: false
        gimbal: false
        liveview: false
        hms: false
        data_transmission: true

      data_frequency: # Options are: 1, 5, 10, 50, 100, 200, 400 Hz
        imu: 100
        attitude: 100
        acceleration: 50
        velocity: 50
        angular_velocity: 100
        position: 50
        altitude: 50
        gps_fused_position: 10
        gps_data: 5
        rtk_data: 5
        magnetometer: 50
        rc_channels_data: 1
        gimbal_data: 1
        flight_status: 1
        battery_level: 1
        control_information: 1
